src/commands/Misc/ship.ts

```diff
@@ -0,0 +1,93 @@
+import { SkyraCommand } from '@lib/structures/SkyraCommand';
+import { CanvasColors } from '@lib/types/constants/Constants';
+import { UserSettings } from '@lib/types/settings/UserSettings';
+import { socialFolder } from '@utils/constants';
+import { fetchAvatar, loadImage, roundNumber } from '@utils/util';
+import { Image } from 'canvas';
+import { Canvas } from 'canvas-constructor';
+import { remove as removeConfusables } from 'confusables';
+import { User } from 'discord.js';
+import { CommandStore, KlasaMessage } from 'klasa';
+import { join } from 'path';
+
+export default class extends SkyraCommand {
+
+	private removeSymbolsRegex = /^(?:[~`!@#%^&*(){}[\];:"'<,.>?/\\|_+=-])+$/g;
+	private lightThemeTemplate: Buffer | null = null;
+	private darkThemeTemplate: Buffer | null = null;
+	private heartIcon: Image | null = null;
+
+	public constructor(store: CommandStore, file: string[], directory: string) {
+		super(store, file, directory, {
+			cooldown: 10,
+			description: language => language.tget('COMMAND_SHIP_DESCRIPTION'),
+			extendedHelp: language => language.tget('COMMAND_SHIP_EXTENDED'),
+			requiredPermissions: ['ATTACH_FILES'],
+			runIn: ['text'],
+			usage: '(firstUser:firstUser) (secondUser:secondUser)',
+			usageDelim: ' '
+		});
+
+		this.createCustomResolver('firstUser', (arg, possible, message) => {
+			if (!arg) return message.guild?.members.random()!.user;
```

Comment:
Won't this `?.` generate code that is..useless? We'll always have message.guild in this context